qiskit/circuit/library/standard_gates/brg.py

```diff
@@ -0,0 +1,57 @@
+"""bridge gate."""
+
+from qiskit.circuit.gate import Gate
+from qiskit.circuit.quantumregister import QuantumRegister
+
+
+class BRGGate(Gate):
+    r"""
+    The Bridge gate.
+
+    This implementation is based on Page 2, Figure 2 of [1], with the definition of
+    Bridge_gate which is composed of 4 CNOTs gates.
+
+    [1] Toshinari et al., https://arxiv.org/pdf/1907.02686.pdf
+
+
+    **Circuit symbol:**
+
+    .. parsed-literal::
+
+        q_0: ──■──
+               │
+        q_1:  ───
+             ┌─┴─┐
+        q_2: ┤ X ├
+             └───┘
+
+
+    ..Note::
+
+    The bridge gate is used as a transformation rules in mapping.
+
+    """
+    # pylint: disable = E1101
+
+    def __init__(self, label=None):
+        """Create new BrG gate."""
+        super().__init__("brg", 3, [])
```

Comment:
Unless there's a need to be more compact, I think we can use the full name here (for the `'name'`, class name, and filename.)